Library/Homebrew/cli_parser.rb

```diff
@@ -16,14 +16,28 @@ def initialize(&block)
         Homebrew.args.instance_eval { undef tap }
         @constraints = []
         @conflicts = []
+        @processed_options = []
+        @desc_line_length = 65
         instance_eval(&block)
+        post_initialize
+      end
+
+      def post_initialize
+        @parser.on_tail("-h", "--help", "Show this message") do
+          puts @parser
+        end
+      end
+
+      def wrap_option_desc(desc)
+        Formatter.wrap(desc, @desc_line_length).split("\n")
```

Comment:
What happens without these being wrapped? Is it the manpage/markdown/something else that looks weird?

Reply:
@GauthamGoli I see, thanks 👍